docs/tutorials/get-started/servo-mousemover.md

```diff
@@ -151,38 +152,39 @@ The servo is now physically connected to the Raspberry Pi, but the Viam app hasn
 
 Go to the [Viam app](https://app.viam.com), and navigate to the **Config** tab.
 
-![This is a screenshot of the Viam app on the CONFIG page. It shows that Builder is chosen as the mode. Below that in the components tab it shows Create Component and to the far left It says Name with a box for a Name to be typed in. The next box to the right says Type and it shows a drop down list of different types of components in Viam. To the far right it says Model, but it's greyed out because none of the previous two fields have been filled in. Directly to the right of Model is a Create Component button that will create the component.](/tutorials/single-component-tutorials-servo-mousemover/new-component.png)
-
 ### Board component
 
-In the **Config** tab, create a new component with the following attributes:
+Create a [board component](/components/board/):
+
+Navigate to the **Components** subtab of your robot's **Config** tab.
 
-- **Name**: `local` - you can name this whatever you want, you will reference it later.
-- **Type**: `board`
-- **Model**: `pi`
+Click **Create component** in the lower-left corner.
 
-![This is a screenshot of the Viam app on the CONFIG page. It shows that Builder is chosen as the mode. Below that in the components tab it shows Create Component and to the far left It says Name with a box and the name local typed in. The next box to the right says Type and it shows a drop down list of different types of components in Viam with board selected. To the far right it says Model with a drop down list and pi is selected. Directly to the right of Model is a Create Component button that will create the component.](/tutorials/single-component-tutorials-servo-mousemover/new-board.png)
+Select `board` for type and `pi` for model.
 
-Click **Create Component** to create the pi [board component](/components/board/) component.
+Enter `local` as the name for your board, then click **Create**.
+
+You can name the board whatever you want.
+You will reference it later.
```

Comment:
What do you think of deleting these lines outright? We are already proposing a value for the board name. For tutorial context, I think it is appropriate to use an example name throughout: `local`.

Same with `FS90R` below.

Reply:
I think that, since the board name is used in configuring the servo but not in the code, that it's fine to keep this note in as a learning moment?

Reply:
Alright that works. I'm just trying to avoid this kind of language:
"this value you entered does not matter"
"but it's important at an unspecified later date, don't forget"

Goal: names are up to you! But for this specific tutorial, use this one in particular
Actual: an aside about naming not mattering in the middle of actionable steps.

Feel free to retain if you find it helpful, I find it potentially convoluting!